File: 02566.py
max_n = 0
for i in range(9):
    a = list(map(int, input().split()))

    max_a = max(a)
    if max_a > max_n:
        max_n = max_a
        max_i = i + 1
        max_j = a.index(max_a) + 1

print(max_n)
print(max_i, max_j, sep=' ')

File: 02566.cpp
#include <iostream>
using namespace std;

int main() {
	int max = 0, max_i, max_j;
	for (int i = 0; i < 9; i++) {
		for (int j = 0; j < 9; j++) {
			int a; cin >> a;
		
			if (a > max) {
				max = a;
				max_i = i;
				max_j = j;
			}
		}
	}

	cout << max << endl;
	cout << max_i + 1 << ' ' << max_j + 1 << endl;
}
